python/mathsys/v3/source/web.sh

```
@@ -0,0 +1,24 @@
+#!/bin/bash
+#^
+#^  COMPILE
+#^
+
+#> COMPILE -> COMMAND
+(
+    cd python/mathsys/dev/source
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)